core/domain/email_manager.py

```diff
@@ -503,3 +503,48 @@ def send_instant_feedback_message_email(
             recipient_id, feconf.SYSTEM_COMMITTER_ID,
             feconf.EMAIL_INTENT_FEEDBACK_MESSAGE_NOTIFICATION, email_subject,
             email_body, feconf.NOREPLY_EMAIL_ADDRESS)
+
+
+def send_report_email(
+        exploration_title, exploration_id, reporter_id, report_type,
+        exploration_owner_ids):
+    email_subject = 'New report for "%s"' % exploration_title
+
+    email_body_template = (
+        'Hello Moderator,<br>'
+        '%s has submitted a new report on the exploration %s of the type %s, '
+        '<a href="https://www.oppia.org/explore/%s">"%s"</a>.<br>'
+        'The author/s of the exploration is/are %s'
+        'You can modify the exploration by clicking '
+        '<a href="https://www.oppia.org/create/%s">"Edit %s"</a>.<br>'
+        '<br>'
+        'Thanks!<br>'
+        '- The Oppia Team<br>'
+        '<br>%s')
+
+    if not feconf.CAN_SEND_EMAILS:
+        log_new_error('This app cannot send emails to users.')
+        return
+
+    if not feconf.CAN_SEND_FEEDBACK_MESSAGE_EMAILS:
```

Comment:
I think we don't need this check, since this is a different category from feedback message emails to authors.